src/app/policies/views/PolicySummaryView.vue

```diff
@@ -100,18 +100,19 @@
 </template>
 
 <script lang="ts" setup>
+import type { Policy } from '../data'
 import DefinitionCard from '@/app/common/DefinitionCard.vue'
 import EmptyBlock from '@/app/common/EmptyBlock.vue'
 import ResourceCodeBlock from '@/app/common/ResourceCodeBlock.vue'
-import type { PolicyEntity, PolicyType } from '@/types/index.d'
+import type { PolicyType } from '@/types/index.d'
```

Comment:
How come this type isn't coming from `../data`, its an API type right?

Reply:
I haven’t added any transformations for this API because we need next to nothing to change on it. The only thing we use in one place (Dataplane policies) is creating a `policyTypesByName` object from `policyTypesData.policies.reduce((obj, policyType) => Object.assign(obj, { [policyType.name]: policyType }), {})`, but I’m not sure we really need to manage this on the data layer.

And as for the type import: this one is used in a bunch of places and I don’t really want to update all the imports right now seeing also that we don’t need them to be different now. There is probably a couple of things that are going to change with regards to type organization going forward, so I’m not too worried about this one being import like that.

Reply:
Could you re-export it from `../data`?

I want `@/types/index.d` to be gone from our application.

If not don't worry, if you could either make the change or not, then feel free to resolve.